RFS/src/main/java/com/rfs/common/OpenSearchClient.java

```diff
@@ -298,6 +342,15 @@ public boolean hasFailedOperations() {
             return matcher.find();
         }
 
+        public List<String> getSuccessfulDocs() {
+            try {
+                return BulkResponseParser.findSuccessDocs(body);
+            } catch (IOException ioe) {
+                log.warn("Unable to process bulk request for success", ioe);
```

Comment:
Should this be at error? It's going to mean the failedRequestsLogger has extra docs in it